#!/usr/bin/env python

PACKAGE = 'coral_usb'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('score_thresh', double_t, 0, 'Threshold for confidence score', 0.60, 0.0, 1.0)
gen.add('patch_width', int_t, 0, 'Width of the image patch', 128, 0, 1024)
gen.add('patch_height', int_t, 0, 'Height of the image patch', 128, 0, 1024)

exit(gen.generate(PACKAGE, PACKAGE, 'EdgeTPUPatchDetector'))
